### PR TITLE
MCO-555 Add AIO variants of init and sysconfig files

### PR DESCRIPTION
**Pull request by richardc**
The All-In-One Agent configures MCollective with a specific set of
paths for configurations and pidfiles.

Here we take the work of Geoff Nichols from PR#274 and moves it
into ext/aio, where the All-In-One build process can consume the
files.

### CODE REVIEW

File: ext/aio/debian/mcollective.init [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+uid=`id -u`
+[ $uid -gt 0 ] && { echo "You need to be root to run file" ; exit 4 ; }
+
+
+# PID directory
+pidfile="/var/run/puppetlabs/agent/mcollectived.pid"
+
+name="mcollective"
+mcollectived=/opt/puppetlabs/agent/bin/mcollectived
```
Comment (geoffnichols) [inline ext/aio/debian/mcollective.init]: @richardc Will there be an update to install.rb to use bin (instead of sbin) for mcollectived? cc/ @haus

Comment (richardc) [inline ext/aio/debian/mcollective.init]: AIO uses install.rb? Then yes I'll add an option to install.rb to use bin for mcollectived.

Comment (richardc) [inline ext/aio/debian/mcollective.init]: Looking at install.rb no modifications are required.

When invoking install.rb add `--sbindir=/opt/puppetlabs/agent/bin`

**Comment by geoffnichols** [comment]
:+1:

**Pull request merged**